.gitignore

```diff
@@ -12,6 +12,10 @@ node_modules
 !/public/assets/modified
 /.docs/.out/*
 
+# Custom modules
+interface/modules/custom_modules/**
+!interface/modules/custom_modules/README.md
```

Comment:
would probably do this instead in a separate commit/PR that also supported a interface/modules/core_modules/ as discussed on saturday call. if have time over next couple days, i may take a stab at that functionality :)

Reply:
just realized you probably have your custom module in there now, so rec just keeping this in place then.